ui/src/main/webapp/wizards/sources/validation.spec.js

```diff
@@ -251,4 +252,43 @@ describe('Sources Validation', () => {
       }), 'Next button should be enabled').to.equal(false)
     })
   })
+
+  describe('Valid URLs', () => {
+    const runs = [
+        {it: 'http://foo.com/blah_blah'},
```

Comment:
I don't think we need to have a list of objects, we can simply use a list of strings instead.